src/ansi.rs

```diff
@@ -360,6 +360,9 @@ pub enum CursorStyle {
 
     /// Cursor is a vertical bar `⎸`
     Beam,
+
+    /// Cursor is a box like `☐`
+    Box,
```

Comment:
This looks new in this PR. Can you pull it out into a separate patch? Small nit also -- it would be nice to name this `Hollow` or `Rect` or something other than `Box`.

Reply:
Or maybe `HollowBlock` would be best